### PR TITLE
chore: add Cloud Engineer role

### PR DESCRIPTION
**Pull request by bsmth**
__Description:__
* Change Cloud Architect role to Cloud Engineer, add relevant skills

### CODE REVIEW

File: src/pages/careers/senior-cloud-engineer.mdx [hunk 1]
```diff
@@ -47,10 +49,15 @@ stack, and infrastructure along the way.
...
 - Experience building scalable, resilient, and secured applications
 - Experience with CI/CD & Infrastructure as Code (e.g., Terraform)
-- Experience with containerized applications (e.g., Docker, Fargate)
-- Experience with automated testing, logging, and monitoring
-- Strong experience managing Linux servers, shell scripting, SSH, etc.
-- Strong experience with load balancing/proxies, network architecture, and
-  security
-
-<CareerEmailUs title="Senior Cloud Architect" />
+- Proficient in Node.js and familiarity with either React or TypeScript
```
Comment (bluestreak01) [inline src/pages/careers/senior-cloud-engineer.mdx:52]:
```suggestion
- Proficient in TypeScript and Node.js. Familiarity with React is a bonus
```

File: src/pages/careers/senior-cloud-engineer.mdx [hunk 2]
```diff
@@ -47,10 +49,15 @@ stack, and infrastructure along the way.
...
+- Experience with containerized applications and orchestration (e.g., Docker,
+  Fargate, Kubernetes)
```
Comment (bluestreak01) [inline src/pages/careers/senior-cloud-engineer.mdx:54]:
```suggestion
Kubernetes)
```

File: src/pages/careers/senior-cloud-engineer.mdx [hunk 3]
```diff
@@ -47,10 +49,15 @@ stack, and infrastructure along the way.
...
+- Experience with automated testing, logging and monitoring
```
Comment (bluestreak01) [inline src/pages/careers/senior-cloud-engineer.mdx:55]:
```suggestion
- Experience with automated testing
```

File: src/pages/careers/senior-cloud-engineer.mdx [hunk 4]
```diff
@@ -47,10 +49,15 @@ stack, and infrastructure along the way.
...
+- Experience with load balancing/proxies, network architecture and security
```
Comment (bluestreak01) [inline src/pages/careers/senior-cloud-engineer.mdx:56]:
```suggestion
- Good understanding of Cloud topology
```

File: src/pages/careers/senior-cloud-engineer.mdx [hunk 5]
```diff
@@ -47,10 +49,15 @@ stack, and infrastructure along the way.
...
+- Experience managing Linux servers, shell scripting, SSH, etc.
```
Comment (bluestreak01) [inline src/pages/careers/senior-cloud-engineer.mdx:57]:
```suggestion
```

if we want a cloud engineer, linux is implied

**Review by bluestreak01 — CHANGES_REQUESTED** [review]

File: src/pages/careers/senior-cloud-engineer.mdx (hunk 5 above)
Comment (bsmth) [reply src/pages/careers/senior-cloud-engineer.mdx]: 👍🏻

**Review by bluestreak01 — APPROVED** [review]

**Pull request merged**